client/src/js/views/DashboardView.js

```diff
@@ -11,7 +11,14 @@ var DashboardView = Backbone.View.extend({
 
   initialize: function() {
     this.render();
-    this.collection = new DogsCollection();
+    this.collection = new DogsCollection(null, {
```

Comment:
@akrawchyk nothing actually. Thomas was helping me figure something out. Leftover artifact from that. Removing